Functional/functional.py

```diff
@@ -0,0 +1,93 @@
+
+"""
+sequential_map - функция должна принимать в качестве аргументов любое количество функций
+(позиционными аргументами, НЕ списком), а также контейнер с какими-то значениями.
+Функция должна возвращать список результатов последовательного применения переданных функций
+к значениям в контейнере. Например,
+sequential_map(np.square, np.sqrt, lambda x: x**3, [1, 2, 3, 4, 5]) -> [1, 8, 27, 64, 125]
+"""
+
+
+def sequential_map(*args):
+    *functions, values_ = args
+    results = values_
+    for func in functions:
+        results = func(results)
+    return results
```

Comment:
Функция должна применять функции ко всем элементам контейнера (как `map`)
```suggestion
    for func in functions:
        results = map(func, results)
    return list(results)
```